sources/packages/web/src/components/students/ApplicationDetails.vue

```diff
@@ -0,0 +1,232 @@
+<template>
+  <v-container>
+    <v-card>
+      <formio
+        formName="studentapplicationdetails"
+        :data="initialData"
+        @customEvent="customEventCallback"
+      ></formio>
+    </v-card>
+    <v-card>
+      <v-container>
+        <p class="category-header-large">Track your application</p>
+        <div class="progress">
+          <div
+            class="progress-bar bg-secondary"
+            :class="getTrackingClass()"
+            role="progressbar"
+            :style="progressBarStyle"
+            aria-valuemax="100"
+          >
+            {{ progressBarLabel }}
```

Comment:
I am not sure if the label in the middle of the progress is adding much value to it.
I would recommend removing it if it does not impact any acceptance criteria.

Reply:
screenshot for the label, 
![image](https://user-images.githubusercontent.com/77353155/134996198-84bce3e9-5d16-414a-ad34-df3cab25764a.png)
.
anyway, this is the bootstrap progress bar, when the vuetify3 stable version is released, we can use it

Reply:
Ohhh yeah, makes sense then. Thanks for clarifying 😉